docs_2.0/manual_EN/3.quick-start/4.nebua-graph-crud.md

```diff
@@ -0,0 +1,474 @@
+
+    ```nGQL
+    INSERT VERTEX <tag> (<property name 1>[, <property name 2>...]) 
+    [, <tag> (<property name 3>[, <property name 4>...]), ...] 
+    {VALUES | VALUE} "<VID>": (<property value 1>[, <property value2>...]);
+    ```
+
+    `VID` is short for vertex ID. It must be a unique int64 value in a graph space.
+
+* Create edges:
+
+    ```nGQL
+    INSERT EDGE <edge type> (<property name 1>[, <property name 2>...]) 
+    {VALUES | VALUE} <source vid> -> <destination vid>[@<rank>] : (<property value 1>[, <property value 2>...]) 
+    [, <source vid> -> <destination vid>[@<rank> : (<property name 1>[, <property name 2>...]), ...]
+    ```
+
+### Examples
+
+* Create vertices representing NBA players and teams:
+
+    ```nGQL
+    (root@nebula) [nba]> INSERT VERTEX player(name, age) VALUES "player100":("Tim Duncan", 42);
+    Execution succeeded (time spent 2919/3485 us)
+
+    Fri, 16 Oct 2020 03:41:00 UTC
+
+    (root@nebula) [nba]> INSERT VERTEX player(name, age) VALUES "player101":("Tony Parker", 36);
+    Execution succeeded (time spent 3007/3539 us)
+
+    Fri, 16 Oct 2020 03:41:58 UTC
+
+    (root@nebula) [nba]> INSERT VERTEX player(name, age) VALUES "player102":("LaMarcus Aldridge", 33);
+    Execution succeeded (time spent 2449/2934 us)
+
+    Fri, 16 Oct 2020 03:42:16 UTC
+
+    (root@nebula) [nba]> (root@nebula) [nba]> INSERT VERTEX team(name) VALUES "team200":("Warriors"), "team201":("Nuggets");
+    Execution succeeded (time spent 3514/4331 us)
+
+    Fri, 16 Oct 2020 03:42:45 UTC
+    ```
+
+* Create edges representing the relations between NBA players and teams:
+
+    ```nGQL
+    (root@nebula) [nba]> INSERT EDGE follow(degree) VALUES "player100" -> "player101":(95);
+    Execution succeeded (time spent 1488/1918 us)
+
+    Wed, 21 Oct 2020 06:57:32 UTC
+
+    (root@nebula) [nba]> INSERT EDGE follow(degree) VALUES "player100" -> "player102":(90);
+    Execution succeeded (time spent 2483/2890 us)
+
+    Wed, 21 Oct 2020 07:05:48 UTC
+
+    (root@nebula) [nba]> INSERT EDGE follow(degree) VALUES "player102" -> "player101":(75);
+    Execution succeeded (time spent 1208/1689 us)
+
+    Wed, 21 Oct 2020 07:07:12 UTC
+
+    (root@nebula) [nba]> INSERT EDGE serve(start_year, end_year) VALUES "player100" -> "team200":(1997, 2016), "player101" -> "team201":(1999,  2018);
+    Execution succeeded (time spent 2170/2651 us)
+
+    Wed, 21 Oct 2020 07:08:59 UTC
+    ```
+
+## Read data
+
+You can use the statements `GO`, `FETCH`, and `LOOKUP` to read vertex and edge data.
+
+|Statement|Description|Syntax|
+|-|-|-|
+|[GO](TODO) [TODO]|The most commonly used read statement. It traverses the database based on specific conditions. A `GO` traversal starts from one or more vertices, along one or more edges, and return information in a form specified in the `GO` command.|`GO [[<M> TO] <N> STEPS ] FROM <vertex list> OVER <edge_type_list> [REVERSELY] [BIDIRECT] [ WHERE <expression> [ AND | OR expression ...]) ] YIELD [DISTINCT] <return_list>`|
```

Comment:
The tabular look is really weird. You can take a preview .

plain text is ok

Reply:
Altered to uls.